R/utilities-figures.R

```diff
@@ -280,62 +284,36 @@ createPlotsFromExcel <- function(
     plotGridNames = NULL,
     simulatedScenarios = NULL,
     observedData = NULL,
+    dataCombinedList = NULL,
     projectConfiguration,
     outputFolder = NULL,
     stopIfNotFound = TRUE) {
   validateIsOfType(observedData, "DataSet", nullAllowed = TRUE)
   validateIsOfType(projectConfiguration, "ProjectConfiguration")
   validateIsString(plotGridNames, nullAllowed = TRUE)
+  validateIsOfType(dataCombinedList, "DataCombined", nullAllowed = TRUE)
 
-  # read sheet "plotGrids" with info for plotGridConfigurations
-  dfPlotGrids <- readExcel(projectConfiguration$plotsFile,
-    sheet = "plotGrids"
+  plotConfigurations <- .readPlotConfigurations(
```

Comment:
Reading of excel sheet exctracted to an internal function to avoid code duplication (used by createPlotsFromExcel and getDataCombinedNames